activerecord/test/models/contact.rb

```diff
@@ -10,14 +10,14 @@ def self.extended(base)
         table_name => "id"
       }
 
-      column :id,             :integer
-      column :name,           :string
-      column :age,            :integer
-      column :avatar,         :binary
-      column :created_at,     :datetime
-      column :awesome,        :boolean
-      column :preferences,    :string
-      column :alternative_id, :integer
+      column :id,             'integer'
+      column :name,           'string'
+      column :age,            'integer'
+      column :avatar,         'binary'
+      column :created_at,     'datetime'
+      column :awesome,        'boolean'
+      column :preferences,    'string'
+      column :alternative_id, 'integer'
```

Comment:
I might be wrong but I believe `sql_type` is never a symbol in real world scenario, so I took the liberty to refactor this test data.